README.md

```diff
@@ -8,19 +8,48 @@ Our open source community's Wiki focuses on answering your questions about learn
 
 ## Get a Fork ready
 
-You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
-Once you get a fork, you will clone it with `git clone url` where `url` is the link to your fork.
+1. Fork it: You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
+2. Once you get a fork, you will clone it with:
 
-Once that is done, you can start contributing by checking our [issues](https://github.com/FreeCodeCamp/wiki/issues) and creating nee pull requests.
+  ```sh
+  git clone https://github.com/YOUR-USERNAME/wiki.git
+  ```
 
-## Contribution Guides
+3. Add the remote: You will need a way to keep your fork syncronized with the original repository.
+
+  ```sh
+  git remote -v
+  origin  https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
+  origin  https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
+  ```
+
+  This means there is no remote pointing to the original one so you would add it with:
+
+  ```sh
+  git remote add upstream https://github.com/FreeCodeCamp/wiki.git
+  ```
+
+  You can verify the changes with `git remote -v`
 
-We currently have a couple of guides to help you contribute, via the browser, command line or desktop application.
+  ```sh
+  git remote -v
+  origin    https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
+  origin    https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
+  upstream  https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (fetch)
+  upstream  https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (push)
+  ```
 
-- [How To Contribute From Your Browser:](Wiki-Contribute-Online) This is currently the easiest way we have to contribute. A simplified version can be found on [here](https://medium.freecodecamp.com/how-to-land-your-first-open-source-contribution-from-your-browser-in-15-minutes-756d9bbf81ad) also.
-- [How To Contribute To The Wiki Locally:](Wiki-Contribute-Local-GUI) This is a guide that will show you how to have a local copy so you can make changes from your computer, it also works for contributing to the main repository.
-- [Pull Request Guidelines](PULL_REQUEST_TEMPLATE)
-- [More on how to make a pull request from a fork](Pull-Request-Contribute)
-- [How to Squash Commits:](git-rebase#squashing-multiple-commits-into-one) We required only one commit per Pull Request unless otherwise specified by a moderator.
+4. Sync the fork when needed: After some other PR has been merged, you will need to fetch the changes and rebase yoru work.
```

Comment:
~~After some other PR has been merged, you will need to fetch the changes and rebase yoru work.~~ After another PR has been merged, you will need to fetch the changes and rebase your work.